### PR TITLE
fix for bug in collectGcBiasMetrics

### LINKED ISSUE #212: CollectGcBiasMetrics java.lang.IndexOutOfBoundsException
`CollectGcBiasMetrics` throws `java.lang.IndexOutOfBoundsException` and I don't see why.

At first, I thought this might be due difference in java versions. The same command worked on one system with java 1.6 and failed on a second system with java 1.7. On the second system, I changed to java 1.6 and the command still failed. So, it's not a java version issue.

Could you provide any clues about how I might fix or work around this error?

The log file for the failed run looks like this:

```
[Thu May 21 16:17:12 EDT 2015] picard.analysis.CollectGcBiasMetrics CHART_OUTPUT=sample1.gc_bias_metrics.pdf SUMMARY_OUTPUT=sample1.gc_bias_metrics_summary INPUT=sample1.bam OUTPUT=sample1.gc_bias_metrics VERBOSITY=DEBUG VALIDATION_STRINGENCY=LENIENT REFERENCE_SEQUENCE=/data/srlab/external-data/iGenomes/Homo_sapiens/UCSC/hg19/Sequence/WholeGenomeFasta/genome.fa    WINDOW_SIZE=100 MINIMUM_GENOME_FRACTION=1.0E-5 IS_BISULFITE_SEQUENCED=false ASSUME_SORTED=true STOP_AFTER=0 QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false
[Thu May 21 16:17:12 EDT 2015] Executing as ks38@rgs10.research.partners.org on Linux 2.6.32-431.29.2.el6.x86_64 amd64; Java HotSpot(TM) 64-Bit Server VM 1.8.0_31-b13; Picard version: 1.130(8b3e8abe25f920f5aa569db482bb999f29cc447b_1427207353) IntelDeflater
[Thu May 21 16:17:12 EDT 2015] picard.analysis.CollectGcBiasMetrics done. Elapsed time: 0.00 minutes.
Runtime.totalMemory()=5145362432
To get help, see http://broadinstitute.github.io/picard/index.html#GettingHelp
Exception in thread "main" java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:653)
	at java.util.ArrayList.get(ArrayList.java:429)
	at java.util.Collections$UnmodifiableList.get(Collections.java:1309)
	at picard.analysis.CollectGcBiasMetrics.setup(CollectGcBiasMetrics.java:110)
	at picard.analysis.SinglePassSamProgram.makeItSo(SinglePassSamProgram.java:98)
	at picard.analysis.SinglePassSamProgram.doWork(SinglePassSamProgram.java:53)
	at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:187)
	at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:95)
	at picard.cmdline.PicardCommandLine.main(PicardCommandLine.java:105)
```

### PR DESCRIPTION
**Pull request by yfarjoun**
Resolves #212 though it would still be good if a test be added, perhaps using user's supplied header.

### CODE REVIEW

File: src/java/picard/analysis/CollectGcBiasMetrics.java [hunk 1]
```diff
@@ -107,7 +107,12 @@ public static void main(final String[] args) {
     protected void setup(final SAMFileHeader header, final File samFile) {
         IOUtil.assertFileIsWritable(CHART_OUTPUT);
         if (SUMMARY_OUTPUT != null) IOUtil.assertFileIsWritable(SUMMARY_OUTPUT);
-        saveHeader = header.getReadGroups().get(0).getLibrary();
+
+        try{
+            saveHeader = header.getReadGroups().get(0).getLibrary();
+        } catch (Exception ignored){
+            saveHeader = "Unavailable";
```
Comment (bradtaylor) [inline src/java/picard/analysis/CollectGcBiasMetrics.java]: Perhaps more specific message, such as "ReadGroupsUnavailable"?

**Comment by bradtaylor** [comment]
One potential comment, otherwise thumbs-up

File: src/java/picard/analysis/CollectGcBiasMetrics.java (hunk 1 above)
Comment (yfarjoun) [inline src/java/picard/analysis/CollectGcBiasMetrics.java]: This is use in the plot as the name of the library....

**Comment by yfarjoun** [comment]
made requested change. waiting for tests.

**Pull request merged**